impeller/fixtures/sample.vert

```diff
@@ -4,7 +4,7 @@
 
 #include "types.h"
 
-uniform UniformBufferObject {
+layout (std140, binding = 17) uniform UniformBufferObject {
```

Comment:
Perhaps specify the set as well?

Reply:
Since the default is zero, it may be easy to miss.

Reply:
done!